lib/index.js

```diff
@@ -21,6 +21,7 @@ class PWMetrics {
     this.url = url;
     this.opts = opts;
     this.opts.disableCpuThrottling = this.opts.disableCpuThrottling || false;
+    this.opts.showChart = (typeof this.opts.showChart !== 'undefined') ? (this.opts.showChart == 'true') : true;
```

Comment:
I see 2 possible solutions here
1. easy one like you did, just simplify a bit: `this.opts.showChart = (this.opts.showChart === 'false') ? false : true;`
2. manage it on `cli.js` level - https://github.com/paulirish/pwmetrics/blob/master/bin/cli.js#L17
```
flags[flagKey] = keyValue[1] || true;
flags[flagKey] = (flags[flagKey] === 'false') ? false : flags[flagKey];
```
I'd prefer second one.